src/libs/actions/Welcome.ts

```diff
@@ -49,13 +60,15 @@ Onyx.connect({
 Onyx.connect({
     key: ONYXKEYS.IS_LOADING_REPORT_DATA,
     initWithStoredValues: false,
-    callback: (val) => {
-        isLoadingReportData = val;
+    callback: (value) => {
+        if (value) {
```

Comment:
why did you add this conditional?

Reply:
ah so in the previous code, we had the possibility that `undefined` or `null` was assigned to `isLoadingReportData` 😅